### PR TITLE
fix: Add security context to pvc.

### PR DESCRIPTION
**Pull request by yogeshg**

### CODE REVIEW

**Review by jsbroks — APPROVED** [review]

File: weave.tf [hunk 1]
```diff
@@ -136,6 +136,11 @@ resource "kubernetes_deployment" "weave" {
             }
           }
         }
+        securityContext {
+          fsGroup = 0
+          runAsUser = 999
+          fsGroupChangePolicy = "OnRootMismatch"
+        }
```
Comment (yogeshg) [inline weave.tf:143]:
```suggestion
        }

```

File: weave.tf [hunk 2]
```diff
@@ -136,6 +136,12 @@ resource "kubernetes_deployment" "weave" {
...
+
```
Comment (yogeshg) [inline weave.tf:144]:
```suggestion
```

**Comment by jsbroks** [comment]
This PR is included in version 1.11.1 :tada:

**Review by vanpelt — COMMENTED** [review]
👍

**Pull request merged**